### PR TITLE
7.5 Remove verification account buttons

### PR DESCRIPTION
**Pull request by Chmelya**
Added hidden views. Removed buttons for verify email, phone numbers and two-factor authorization.
Don't touched any logic.

### CODE REVIEW

File: src/Teams.Web/Areas/Identity/IdentityHostingStartup.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+using Teams.Data;
+
+[assembly: HostingStartup(typeof(Teams.Web.Areas.Identity.IdentityHostingStartup))]
+namespace Teams.Web.Areas.Identity
+{
+    public class IdentityHostingStartup : IHostingStartup
+    {
+        public void Configure(IWebHostBuilder builder)
+        {
+            builder.ConfigureServices((context, services) => {
```
Comment (SergeyMokin) [inline src/Teams.Web/Areas/Identity/IdentityHostingStartup.cs:17]: single line?

**Review by SergeyMokin — CHANGES_REQUESTED** [review]
Did you check, is it possible to configure it using setup properties?

File: src/Teams.Web/Areas/Identity/IdentityHostingStartup.cs (hunk 1 above)
Comment (Chmelya) [reply src/Teams.Web/Areas/Identity/IdentityHostingStartup.cs:17]: generated code

Comment (SergeyMokin) [reply src/Teams.Web/Areas/Identity/IdentityHostingStartup.cs:17]: alright

**Review by SergeyMokin — APPROVED** [review]

**Review by sub13 — APPROVED** [review]

**Review by desenzo — APPROVED** [review]

**Pull request merged**